### DISCUSSION TITLE
suggestion: JQ tester & REGEX tester

### DISCUSSION THREAD

**Discussion by robin-collins** [CorentinTh/it-tools#273] [Ideas]
Firstly, my compliments on a great set of tools.

I would love to see a JQ tester and Regex tester. An example of such tools is

https://regex101.com/
https://www.devtoolsdaily.com/jq_playground/
https://jqkungfu.com/

https://github.com/fiatjaf/awesome-jq

**Comment by scottd34** [comment]
The regex one would be great.  Another similar example is regexr.com.

**Comment by sharevb** [comment]
Hi @robin-collins and @scottd34, Regex implemented in #1030

**Comment by sharevb** [comment]
Hi @robin-collins and @scottd34, JQ/JSONPath implemented in #1262

**Comment by sharevb** [comment]
Hi @CorentinTh, so both implemented in #1262 and #1030 (most awaited)

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
